### DISCUSSION TITLE
How to make advanced text styles?

### DISCUSSION THREAD

**Discussion by nurettn** [fabricjs/fabric.js#10162] [Q&A]
I want to generate advanced test styles like in the Canva

Do you have any idea how to make these and make them editable in the fabricjs canvas?

![image](https://github.com/user-attachments/assets/7d72ce8f-c7fe-4ecb-82e9-1e03a7ec456d)

**Comment by asturur** [comment]
Those are custom fonts or text with shadow or layer of the same text replicated with an offset.
You can already do them.

If you want to make it a single editable object you will have to play with the render method of text to render multiple times, in the case of SALE and SHINE
